pkg/batcher/batcher_test.go

```
@@ -230,3 +231,31 @@ func TestFlushOnClose(t *testing.T) {
+			processed.Add(int32(len(items)))
+
+			return nil
+		}),
+	)
+
+	// ACT
+	for range 10 {
+		b.Add(test.BatchItem{})
+	}
+
+	b.Join(100 * time.Millisecond)
```

golangci-lint (1.64.8)

Error return value of `b.Join` is not checked

(errcheck)